descriptive_statistics.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = 'descriptive_statistics'
-  s.version     = '2.5.1'
+  s.version     = '2.5.2'
```

Comment:
I'm only seeing development changes, not changes to the code of the gem itself, and so not version bump in warranted.

Reply:
ok, usually I do that in the patch to track any changes like updates and stuff so people can reference the new version or hold to the last they had running to avoid unwanted gem update at all. but if you prefere I can roll back the version.